src/credentials/examples/DeviceAttestationCredsExample.cpp

```diff
@@ -21,12 +21,21 @@
 #include <lib/core/CHIPError.h>
 #include <lib/support/Span.h>
 
+#if CHIP_CRYPTO_HSM
+#include <crypto/hsm/CHIPCryptoPALHsm.h>
+#endif
+
+#ifdef ENABLE_HSM_DEVICE_ATTESTATION
+#define DEV_ATTESTATION_KEY 0xDADADADA
+#endif
+
```

Comment:
The DeviceAttestationCredsExample is just an example, and is not meant to have every platform add their feature to it. Please create your own `DeviceAttestationCredentialsProvider` under `platform/nxp/` somewhere and set it in your examples. The specific policy here is 100% NXP (e.g. the DA key ID is 0xDADADADA) and this will be a problem to override or support different HW-assisted attestation.

Have the correct `SetDeviceAttestationCredentialsProvider()` call in your own examples/platform to support the underlying implementation.

Reply:
Created platform specific files - examples\platform\nxp\se05x\DeviceAttestationSe05xCredsExample.cpp